src/main/components/assessment/assessment-dropInfo-tooltip.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component, ComponentClass } from 'react';
+import { IAssessment } from '../../services/assessment/assessment';
+import { Button } from '../controls/button/button';
+import { LinkIcon } from '../controls/icon/link';
+import { TrashIcon } from '../controls/icon/trash';
+import { I18nContext } from '../i18n/i18n-context';
+import ReactTooltip from 'react-tooltip';
+import { AssessmentRepository } from '../../services/assessment/assessment-repository';
+import { compose } from 'redux';
+import { localized } from '../i18n/localized';
+import { connect } from 'react-redux';
+import { ModelState } from '../store/model-state';
+import { IUMLElement } from '../../services/uml-element/uml-element';
+
+type OwnProps = {
+  assessment: IAssessment | null;
```

Comment:
are you sure you want to use `null` and not `undefined`?

Reply:
I just did similar thing with this: https://github.com/ls1intum/Apollon/blob/069e448d6fe14ae1914f9f56927f5284dcbd46f9/src/main/components/assessment/assessment-section.tsx#L49

what do you suggest?